### ISSUE TITLE
smallrye-rabbitmq: Can't publish directly to queue with amq.default (or empty) exchange

### ISSUE THREAD

**Issue by gbzarelli** [smallrye/smallrye-reactive-messaging#1641]
The rabbitmq allow directly publish using the default exchange (amq.default)

![image](https://user-images.githubusercontent.com/6283514/156613411-c4735b08-5690-48db-a160-2d6a338b8044.png)

I configure my quarkus app with the following values:

```propertie
mp.messaging.outgoing.sample-queue.connector=smallrye-rabbitmq
mp.messaging.outgoing.sample-queue.default-routing-key=sample-queue
mp.messaging.outgoing.sample-queue.exchange.name=amq.default
mp.messaging.outgoing.sample-queue.exchange.declare=false
```

Code:
``` java
import org.eclipse.microprofile.reactive.messaging.Channel;
import org.eclipse.microprofile.reactive.messaging.Emitter;
...
    @Channel("sample-queue")
    Emitter<OutputMessage> sampleQueue;
...
    sampleQueue.send(message);
```

But the message **was not reached to destine** and the output log says:

Log DEBUG:
```log
DEBUG [io.sma.rea.mes.rabbitmq] (vert.x-eventloop-thread-0) SRMSG17022: Sending a message to exchange `amq.default` with routing key sample-queue
```

If I publish manually using the same arguments the message was reached:

![image](https://user-images.githubusercontent.com/6283514/156614370-7d7dfc69-3c99-4467-83fc-ab62ff4edd3b.png)

Is it possible to send a direct message from this library? I don't have any problem using exchange, just using the direct publication with the amq.default exchange.

I'm using Quarkus version `2.7.3.Final`  with dependency:
```xml
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-smallrye-reactive-messaging-rabbitmq</artifactId>
    </dependency>
```

**Comment by hjklapp** [comment]
With version 3.16 of the plugin, using `amq.*` as exchange name is forbidden (which is not a problem, as it was not working anyway).

And quarkus config does not allow to set the exchange name to the empty string.

@cescoffier 
Could we specify a special string for the exchange name, such as `default-exchange`, which would be converted to the empty string before establishing the connection to the exchange via vertx?

My hope is that a small change around this line might do the trick.
https://github.com/smallrye/smallrye-reactive-messaging/blob/f92edda52d6e735801eb603823e0dc90c0a643e9/smallrye-reactive-messaging-rabbitmq/src/main/java/io/smallrye/reactive/messaging/rabbitmq/RabbitMQConnector.java#L438

**Comment by ozangunalp** [comment]
Fixed in #1949
